packages/blade/src/components/Checkbox/CheckboxLabel.native.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import type { CheckboxLabelProps } from './CheckboxLabel';
+
+const StyledCheckboxLabel = styled.Pressable({
```

Comment:
the label will be clickable here?

Reply:
Yes. https://github.com/razorpay/blade/pull/544#discussion_r926316645